index.html

```diff
@@ -21,45 +21,70 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="desktop.css">
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <section class="wrap-section">
+    <section class="wrapper">
         <header>
             <nav>
-                <a href="#" class="my-logo">Aneal Laryea</a>
-                <a href="#"><img src="assets/image/menn1.png" alt="menu"></a>
+                 <a href="#" class="my-logo">Aneal Laryea</a>
+                <!--<a href="#"><img src="assets/image/menn1.png" alt="menu"></a> -->
+                <!-- <div class="logo">
+                    <a href="#" class="my-logo">Aneal Laryea</a>
+                  </div> -->
+                
+                <ul class="sects">
+                    <li class="port"><a href="#">Portfolio</a></li>
+                    <li class="aabout mp dbx"><a href="#">About</a></li>
+                    <li class="contact mp dbx"><a href="#">Contact</a></li>
```

Comment:
There is a design issue with the navigation on the desktop view. Moreover kindly use the same font color for navigation items as in the design which is `Inter`.
![nav](https://user-images.githubusercontent.com/35504149/172961978-06939fba-730f-413d-b90e-6c0a73445eb2.png)

Reply:
* Design issue fix. The same font color was used and the same font family was applied to navigation items.